content/guides/references/trade-offs.md

```diff
@@ -267,3 +269,139 @@ app.listen(8081, () => {})
 This avoids ever needing a second open browser, but still gives you an
 end-to-end test that provides 100% confidence that the two clients can
 communicate with each other.
+
+### Navigation Rules
+
+By itself, Cypress limits each test to only visiting domains that are considered
+to be same superdomain to one another. If a navigation occurs that does not meet
+the same superdomain rule, the [`cy.origin`](/api/commands/origin) command must
+be used to execute assertions inside the newly navigated origin.
+
+But what is same superdomain? It is actually very similar to that of same
```

Comment:
I think the rest of this section should be moved to a Cross-Origin Testing guide and linked to.